test/test_helpers.bash

```diff
@@ -22,7 +22,7 @@ setup_asdf_dir() {
   ASDF_BIN="$(dirname "$BATS_TEST_DIRNAME")/bin"
 
   # shellcheck disable=SC2031
-  PATH="$ASDF_BIN:$ASDF_DIR/shims:$PATH"
```

Comment:
@hyperupcall This change again. Am I incorrect, or would the quotes end up in the PATH?

Reply:
The quotes won't end up in the PATH, but it's just bad hygiene I'll fix that